test/IISIntegration.IISServerFunctionalTests/FeatureCollectionTests.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IIS.FunctionalTests;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Server.IIS.FunctionalTests
+{
+    public class FeatureCollectionTest : LoggedTest
+    {
+        public FeatureCollectionTest(ITestOutputHelper output) : base(output)
```

Comment:
Going to disable tests until is fixed (or we build ANCM from source) https://github.com/aspnet/IISIntegration/issues/424